lib/ckb/utils.rb

```diff
@@ -1,8 +1,8 @@
 require 'secp256k1'
 
 module Ckb
-  MIN_CELL_CAPACITY = 40
-  MIN_UDT_CELL_CAPACITY = 48
+  MIN_CELL_CAPACITY = 40 * 10**8
```

Comment:
Can we use the utility function here as well?

Reply:
OK

Reply:
updated, and move to bottom after `Utils` defined